### PR TITLE
Added hasNext helper for status (with tests)

### PR DESCRIPTION
**Pull request by dodsky**
Exported RESOLVED_ENDPOINT constant

### CODE REVIEW

File: test/status.spec.js [hunk 1]
```diff
@@ -426,4 +427,39 @@ describe('Status metadata', () => {
...
+        );
+        const obj = {};
+        setStatus(obj, status);
+
+        expect(hasNext(obj)).to.be.true;
+      });
+
+
+      it('Returns false when next link doesn\'t exist', () => {
+        const status = updateStatus(
```
Comment (MrBr) [inline test/status.spec.js]: I would just add here a case when there is no `links` object at all in the status.

Comment (dodsky) [inline test/status.spec.js]: I've added the missing test

File: src/status.js [hunk 2]
```diff
@@ -50,7 +50,11 @@ export const cloneStatus = (sourceObject, destinationObject, markChange = false)
 };
 
 function statusProp(obj, prop) {
-  return _.get(obj, [STATUS, prop]);
+  if (typeof prop === 'string') {
+    // eslint-disable-next-line no-param-reassign
+    prop = [prop];
+  }
+  return _.get(obj, [STATUS, ...prop]);
```
Comment (MrBr) [inline src/status.js]: I would rather write this like
```
[STATUS, _.isArray(prop) ? ...prop : prop ]
```
 or 
```
const propPath = _.isArray(prop) ? prop : [prop];
return _.get(obj, [STATUS, ...propPath]);
```

File: src/status.js [hunk 3]
```diff
@@ -50,7 +50,8 @@ export const cloneStatus = (sourceObject, destinationObject, markChange = false)
 };
 
 function statusProp(obj, prop) {
-  return _.get(obj, [STATUS, prop]);
+  const propPath = _.isArray(prop) ? prop : [prop];
```
Comment (vedrani) [inline src/status.js]: Why do you need it?

**Pull request merged**